client/assets/net.veloren.airshipper.metainfo.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2021 Adam Blanchet-->
+<component type="desktop-application">
+  <id>net.veloren.airshipper</id>
+  
+  <name>Airshipper</name>
+  <summary>Airshipper is the official launcher for Veloren.</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+
+  <content_rating type="oars-1.0">
+    <content_attribute id="violence-cartoon">moderate</content_attribute>
+    <content_attribute id="violence-fantasy">moderate</content_attribute>
+    <content_attribute id="language-profanity">mild</content_attribute>
+    <content_attribute id="language-humor">mild</content_attribute>
+    <content_attribute id="social-chat">moderate</content_attribute>
+  </content_rating>
+
+  <releases>
+    <release version="0.4.2" date="2020-12-16"/>
+  </releases>
+  
+  <description>
+    <p>
+      Airshipper is the official launcher for Veloren, the online voxel-based RPG.
+    </p>
+    <p>
+      Veloren is a multiplayer open world, voxel-based RPG written in Rust. It is inspired
+      by games such as Cube World, Minecraft, Dwarf Fortress and Breath of the Wild.
+    </p>
+    <p>
+      Veloren's development mainly happens on the official Discord server (https://discord.gg/kjwJwjK).
+      New contributors are always welcome!
+    </p>
+  </description>
+  
+  <launchable type="desktop-id">net.veloren.airshipper.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://camo.githubusercontent.com/71dfc8bb095129c57a7d2c29ff7d50bba4c91e67fef84c2e6ef93be7efb1e02a/68747470733a2f2f7777772e736f6e6774726f6e69782e636f6d2f616972736869707065722d302e342e302e676966</image>
```

Comment:
Also could be forgotten to be updated. Maybe we can document that in [`WORKFLOW.md`](https://github.com/Songtronix/Airshipper/blob/staging/WORKFLOW.md)